packages/contracts-bedrock/contracts/L1/SystemConfig.sol

```diff
@@ -207,4 +217,15 @@ contract SystemConfig is OwnableUpgradeable, Semver {
             sstore(slot, _unsafeBlockSigner)
         }
     }
+
+    /**
+     * @notice Returns the minimum gas limit allowed by the system. If the L2
+     *         gas limit is set to a value smaller than this, then it is
+     *         possible for a block to be produced that uses more gas than what
+     *         is allowed on L2, resulting in a liveness failure.
+     */
+    function minimumGasLimit() public view returns (uint256) {
+        uint256 maxResourceLimit = uint256(ResourceMetering(PORTAL).MAX_RESOURCE_LIMIT());
```

Comment:
I think this needs a comment to explain what's going on, i.e. the relationship between the minimum gas limit here, and the maximum resource limit in the portal.